ephios/extra/context.py

```diff
@@ -1,7 +1,16 @@
 from django.templatetags.static import static
 from django.utils.translation import get_language
 
-from ephios.helpers.signals import footer_link
+from ephios.extra.signals import footer_link
+
+import subprocess
+
+try:
+    EPHIOS_VERSION = (
+        subprocess.check_output(["git", "rev-parse", "--short", "HEAD"]).decode().strip()
```

Comment:
Can we cache that so we don't need to wait for the subproccess on every page load?

Reply:
It's already only done at module load?!